elixir/apps/web/lib/web/router.ex

```diff
@@ -89,6 +89,10 @@ defmodule Web.Router do
   scope "/:account_id_or_slug", Web do
     pipe_through [:browser, :account]
 
+    live_session :client_redirect, on_mount: [Web.Sandbox] do
+      live "/signin_success", SignIn.Success
```

Comment:
We use `sign_in` in URL's below so maybe for consistency, we want something path to be something like `sign_in/success_redirect`

Reply:
Sounds good I can update it